ui/selectors/selectors.js

```diff
@@ -113,6 +113,8 @@ import { hasTransactionData } from '../../shared/modules/transaction.utils';
 import { toChecksumHexAddress } from '../../shared/modules/hexstring-utils';
 import { createDeepEqualSelector } from '../../shared/modules/selectors/util';
 import { isSnapIgnoredInProd } from '../helpers/utils/snaps';
+// eslint-disable-next-line import/no-restricted-paths
+import { getManifestFlags } from '../../app/scripts/lib/manifestFlags';
```

Comment:
Instead of adding an `eslint-disable-next-line` maybe we should move this file to `/shared/lib/`?

Reply:
Good idea, added 35fe1f03a900b79e7400f84dc48a7afe05f19ffd and some refactor is included